airflow/contrib/operators/qubole_operator.py

```diff
@@ -160,6 +161,9 @@ def get_hook(self):
         # Reinitiating the hook, as some template fields might have changed
         return QuboleHook(*self.args, **self.kwargs)
 
+    def get_redirect_url(self, dttm, redirect_to):
+        return self.get_hook().get_redirect_url(self, dttm)
```

Comment:
the `get_redirect_url` in qubole hook has task argument?

Reply:
Yes. It pulls from xcom, which is task based, so it needs the task instance to do that.